cnames_active.js

```diff
@@ -507,6 +507,7 @@ var cnames_active = {
   "fuck": "gerd2002.github.io/fuck.js",
   "funbook": "egoist.github.io/funbook",
   "funfuncs": "floffah.github.io/funfuncs",
+  "fucao": " fusuhub.github.io",
```

Comment:
```suggestion
  "fucao": "fusuhub.github.io",
```
Random white space will need to be removed.

Reply:
> 建议更改Beta 
> [给我们反馈](https://www.research.net/r/SuggestedChanges)
>   “ fucao ”： “  fusuhub.github.io ”，
>   “ fucao ”： “ fusuhub.github.io ”，
> 申请建议 
> 需要删除随机空白区域。

我非常抱歉！我不小心......抱歉。